src/main/java/javax/validation/ElementKind.java

```diff
@@ -52,5 +53,12 @@
 	/**
 	 * The return value of a method or constructor.
 	 */
-	RETURN_VALUE
+	RETURN_VALUE,
+
+	/**
+	 * A container element e.g. a type argument of a generic declaration.
```

Comment:
Technically it's not the type argument itself, but the element within a container:

> An element stored in a container, e.g. the value wrapped by `Optional`, a value stored in a `List` or a value stored in an array.